libraries/matrix/impl/src/main/kotlin/io/element/android/libraries/matrix/impl/room/JoinedRustRoom.kt

```diff
@@ -237,7 +238,7 @@ class JoinedRustRoom(
                     filter = filter,
                     internalIdPrefix = internalIdPrefix,
                     dateDividerMode = dateDividerMode,
-                    trackReadReceipts = trackReadReceipts,
+                    trackReadReceipts = if (trackReadReceipts) TimelineReadReceiptTracking.ALL_EVENTS else TimelineReadReceiptTracking.DISABLED,
```

Comment:
Are we sure about the usage here? What `MESSAGE_LIKE_EVENTS` is about?

Reply:
It's about having the iOS behaviour where read receipts are only added to message like events, instead of any event in the timeline. It's also supposedly way slower than our current behaviour, so I believe we should keep it like this and maybe test the new behaviour in a nightly version instead.